### ISSUE TITLE
Low battery life 3 - 4 hours instead of 9+

### ISSUE THREAD

**Issue by JonnoSA** [gnodipac886/MatebookXPro-hackintosh#96]
Good day all, 

I recently installed macOS Catalina on my 2018 Matebook X Pro with relative ease and everything seems to be working quite well except for the battery life, which can just stretch to 4.5 hours with no apps open. I have tried everything from battery saving apps to trying to undervolt(which I was unable to do due to an error that I could not fix) but nothing seems to be bringing the battery life anywhere close to the advertised 9 - 10+ hours gnodipac886 claims to be getting in his readme. 
I would be very grateful to anyone who could assist with this problem or who knows what to do to increase the battery life.

Many thanks

Jonathan

**Comment by jonescamilla** [comment]
Hey Jonathan,
What issue did you face trying to undervolt?

**Comment by JonnoSA** [comment]
Hi jonescamilla,

I tried to follow this guide for undervolting using the voltageshift method.
https://hackintoshpro.com/undervolt-mac/

When running the "./voltageshift info" command I got the error: "zsh: killed     ./voltageshift info"
![Screenshot 2020-03-21 at 22 50 37](https://user-images.githubusercontent.com/24739247/77236347-64610d00-6bc6-11ea-907b-7ee2c55445dd.png)

I have looked everywhere for a workaround or solution but am not able to find any.
I'm not sure if there is something I am missing or what?

**Comment by jonescamilla** [comment]
Did you give it root permission?
`sudo crown -R root:wheel VoltageShift.kext`

**Comment by JonnoSA** [comment]
It says crown command not found, do you mean chown or chmod?

**Comment by JonnoSA** [comment]
Even after updating permissions it still doesn't seem to be working.
![Screenshot 2020-03-21 at 23 12 16](https://user-images.githubusercontent.com/24739247/77236670-6bd5e580-6bc9-11ea-8a95-a6d0aa065b3b.png)

**Comment by Chatbox1024** [comment]
Man, you need to load the kext.  RTFM:
https://github.com/sicreative/VoltageShift

**Comment by Chatbox1024** [comment]
Here: https://github.com/gnodipac886/MatebookXPro-hackintosh/issues/84
and
here: https://github.com/gnodipac886/MatebookXPro-hackintosh/issues/89

**Comment by JonnoSA** [comment]
Yeah I have been trying to build the kext but I am getting this error now, even with Xcode freshly installed.
<img width="815" alt="Screenshot 2020-03-22 at 09 15 04" src="https://user-images.githubusercontent.com/24739247/77244480-d3b61b80-6c1d-11ea-8a1c-236c74faf96c.png">
